enhancements/windows-containers/container-runtime-containerd.md

```diff
@@ -106,6 +104,12 @@ that doesn't have Docker and containerd will be the only runtime. If the Machine
 Docker, containerd will run along with Docker service. But Kubernetes will use only containerd. Customers can stop the
 Docker service if it is no longer needed. Even if Docker service gets started at any time, it can co-exist with
 containerd.
+Microsoft features Containers, Hyper-V and Hyper-V-PowerShell must be installed for any Windows server to support
+containers. From Windows server 2022 onward, Microsoft is not planning to enable them by default. WMCO needs to take
+care of this during Windows instance bootstrap process. After enabling required services, WMCO must reboot Windows
+instance and then continue bootstrap process. In Windows server 2019 OneGet command takes care of enabling required
```

Comment:
I think OneGet docker installation will continue to be part of 2019. Wanted to mention it here because when we upgrade WMCO, there may be chances that we may not have Containers feature enabled(possible with new image). So we needed to take care of this.